cosmic-settings/src/pages/desktop/appearance/mod.rs

```diff
@@ -2090,8 +2091,11 @@ pub fn interface_density() -> Section<crate::pages::Message> {
         .descriptions(descriptions)
         .view::<Page>(move |_binder, _page, section| {
             let descriptions = &section.descriptions;
-
-            let density = cosmic::config::interface_density();
+            let density = CosmicTk::config()
+                .ok()
+                .unwrap()
+                .get("interface_density")
+                .unwrap();
```

Comment:
Instead of fetching this in every view update (very costly), you can use a cosmic-config subscription to watch for config updates to this file.